src/Components/CommentAdder.js

```diff
@@ -0,0 +1,57 @@
+import { useContext } from "react";
+import { UserContext } from "../contexts/User";
+import { useState } from "react";
+import { postArticleComment } from "../api";
+import "./CommentAdder.css";
+
+const CommentAdder = ({ article_id, setComments }) => {
+  const {user} = useContext(UserContext);
+
+  const [newComment, setNewComment] = useState({
+    username: user.username,
+    body: undefined,
+  });
+
+  const handleChange = (event) => {
+    setNewComment((currentNewComment) => {
+      return { ...currentNewComment, body: event.target.value };
+    });
+  };
+
+  const postSuccess = () => {
+    alert("Comment posted successfully!");
```

Comment:
I would avoid alert as these arent very UI friendly, could we conditionally render a component instead?